src/gui/dialog/PageTemplateDialog.cpp

```diff
@@ -158,8 +156,6 @@ void PageTemplateDialog::showPageSizeDialog() {
 
         updatePageSize();
     }
-
-    delete dlg;
```

Comment:
It looks like dlg is still being allocated with 'new' on line 147... Who is responsible for freeing dlg?

Reply:
Nice catch, thanks.